src/SFA.DAS.RoATPService.Application.Api/Controllers/OrganisationController.cs

```diff
@@ -45,14 +45,15 @@ public async Task<IActionResult> Get(Guid organisationId)
         [SwaggerResponse((int)HttpStatusCode.OK, Type = typeof(IEnumerable<Engagement>))]
         [SwaggerResponse((int)HttpStatusCode.BadRequest, typeof(IEnumerable<Engagement>))]
         [SwaggerResponse((int)HttpStatusCode.InternalServerError, Type = typeof(ApiResponse))]
-        public async Task<IActionResult> GetEngagements()
+        public async Task<IActionResult> GetEngagements(long sinceEventId = 0, int pageSize = 1000, int pageNumber = 1)
```

Comment:
Will these values come in via a Query string? I noticed that there is a GetEngagementsRequest which could potentially be used instead. The GetRegisterStatus has a similar request object. Has it been agreed which is the best approach to use?

Reply:
Yes. The values come from a query string.
GetOrganisationRegisterStatusRequest object is used in GetRegisterStatus, which has different functionality.
 GetEngagementsRequest object is the one used in GetEngagements and the one I have modified to hold the needed properties (SinceEventId, PageSize, PageNumber).
And I have used the three parameters  (sinceEventId, pageSize, pageNumber) instead GetEngagementsRequest object in GetEngagements signature, because that was how is done in example I was asked to follow:
 https://github.com/SkillsFundingAgency/das-providerevents/blob/master/src/api/Submissions_README.md

Reply:
GetEngagements is required for the FCS team, and is not used internally but RoATP (as far as I am aware). However, if it is we do not want to change the way in which the base API endpoint works.